packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -348,6 +349,57 @@ const Breakouts = WebexPlugin.extend({
       },
     });
   },
+
+  /**
+   * Create new breakout sessions
+   * @param {object} sessions -- breakout session group
+   * @returns {Promise}
+   */
+  async create(sessions) {
+    // @ts-ignore
+    const breakInfo = await this.webex.request({
```

Comment:
Both the create and the delete can fail if someone else is editing at the same time. The server response with 
```
{"errorCode":201409024,"message":"Edit lock token mismatch","errors":[{"errorCode":201409024,"description":"Edit lock token mismatch"}],"trackingId":"webex-js-sdk_7b6a0918-5eaf-493e-a989-40073c2b38ea_56"}
```

we should handle this case and return an error

Reply:
add catch and throw error